projects/documentation/content/deprecation.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: deprecation.njk
+title: 'Deprecation Guide: Spectrum Web Components'
+displayName: Deprecation Guide
+slug: deprecation-guide
+---
+
+# Deprecation Guidelines
+
+The deprecation strategy for Spectrum Web Components (SWC) outlines the process for identifying, communicating, and removing deprecated components from the library. This document provides guidelines for managing deprecation and ensuring a smooth transition for users.
+
+## Deprecation Timeline
+
+By default the removal of functionality or a component is considered for the next major release after the major release for which it was marked as deprecated. For example, a function marked as deprecated for the SWC 1.0.0 release will be removed for the SWC 2.0.0 release.
+
+## Deprecation Policy
+
+### Warning Levels
+
+Deprecation warnings are categorised into the following levels:
+
+-   Yellow: Components are marked as deprecated but still functional. Users are encouraged to migrate to alternative solutions.
+
+<img src="https://raw.githubusercontent.com/adobe/spectrum-web-components/98f9b338444fda239650d582d19879c7c72e04e4/.github/images/yellow-warning.png" alt="Yellow warning" width="50%"/>
```

Comment:
I'm not sure that pointing to images hosted by github from the doc site is a good idea. Can we include these in the build process for the docs site?

Reply:
I currently get this: 
<img width="345" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/1156657/e54ff429-2704-40f6-8569-fb76f9a22d89">